src/sdk/pynni/nni/compression/torch/compressor.py

```diff
@@ -143,7 +144,31 @@ def _expand_config_op_types(self, config):
                 expanded_op_types.append(op_type)
         return expanded_op_types
 
-
+class Wrapper(torch.nn.Module):
+    def __init__(self, module, layer, config, pruner):
```

Comment:
It appears layer.module is same as module, so no need to use 2 parameters.

Reply:
Yes, I remove layer parameter for simplicity.